server/views/account/reset.jade

```diff
@@ -7,11 +7,25 @@ block content
       input(type='hidden', name='_csrf', value=_csrf)
       .form-group
         label(for='password') New Password
-        input.form-control(type='password', name='password', value='', placeholder='New password', autofocus=true)
+        input.form-control(type='password', id='password', required, name='password', value='', placeholder='New password', autofocus=true)
       .form-group
         label(for='confirm') Confirm Password
-        input.form-control(type='password', name='confirm', value='', placeholder='Confirm password')
+        input.form-control(type='password', id='confirm-password', required, name='confirm', value='', placeholder='Confirm password')
       .form-group
         button.btn.btn-primary.btn-reset(type='submit')
           i.fa.fa-keyboard-o
           | Change Password
+
+        script.
```

Comment:
Move this line and everything below it (the whole script block), to its correct indentation. Your script block is going to be within the form elements without that. Most browsers will not execute such code.

You should have it rather at the same level as line 4 in this snippet.

This is how the jade(pug) templating engine renders elements, and hence indentation is very crucial.